src/code/z_scene.c

```diff
@@ -9,29 +9,158 @@ GLOBAL_ASM("./asm/nonmatching/z_scene/Scene_ReloadUnloadedObjects.asm")
 
 GLOBAL_ASM("./asm/nonmatching/z_scene/Scene_FindSceneObjectIndex.asm")
 
-GLOBAL_ASM("./asm/nonmatching/z_scene/Scene_IsObjectLoaded.asm")
+// func_8012F668 
+s32 Scene_IsObjectLoaded(SceneContext* iParm1, s32 index) {
+	if (iParm1->objects[index].id > 0)
```

Comment:
Always use spaces, looks like a tab snuck in.

Also, the currently style of this project is to always use braces, even for single-line if statements. This might change once we standardize with oot, but this is the style for now.

Reply:
Good, I prefer this style.